src/components/ArticlesPage.js

```diff
@@ -15,13 +16,35 @@ const styles = StyleSheet.create({
 });
 
 export default class ArticlesPage extends Component { // eslint-disable-line
+  constructor(props) {
+    super(props)
+
+    const ds = new ListView.DataSource({
+      rowHasChanged: (r1, r2) => r1 !== r2,
+    })
+
+    this.state = {
+      articlesDataSource: ds.cloneWithRows(props.articles.toJS()),
```

Comment:
Check out [examples of `ListView`](http://facebook.github.io/react-native/releases/0.42/docs/listview.html#listview). Ask if something is still unclear.